### DISCUSSION TITLE
Data Importer: "Duplicate transactions" after deleting all transactions in Firefly III

### DISCUSSION THREAD

**Discussion by dekiesel** [firefly-iii/firefly-iii#6143] [Firefly III] [answered]
Hi,

I set up my accounts in F3, I then imported a few transactions but realized I made a mistake. So I deleted all transactions in F3. I can't see them and the balances are correct. 

When I now use the data importer to import the same transactions again it errors out every single transaction as a duplicate.

How do I correctly remove those transactions from F3? I can't find them at all so my only guess is that I have to delete them from the DB manually.

Advice is greatly appreciated!

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/data-importer/help/faq/#why-cant-i-import-duplicate-transactions

**Reply by dekiesel** [reply]
Thank you very much!

**Reply by dekiesel** [reply]
Because I'll surely have this same question in a few weeks:

This is the script I used to delete ALL transactions (I am using mariadb):

```
delete from transaction_journals where id>1;
delete from budget_transaction where id>1 ;
delete from budget_transaction_journal where id>1;
delete from category_transaction where id>1;
delete from category_transaction_journal where id>1;
delete from transactions where id>1;
delete from transaction_journals where id>1;
delete from transaction_groups where id>1;
delete from journal_links where id>1;
delete from journal_meta where id>1;
delete from tag_transaction_journal where id>1;
```
